### ISSUE TITLE
Provide Convert for the existing BH.oM.DeepLearning.Activations

### ISSUE THREAD

**Issue by epignatelli** [BHoM/TensorFlow_Toolkit#10]
Check https://github.com/BHoM/Keras_Toolkit/projects/1 to view the status of the converts.
This issue aims at ticking all the Activations column.

- [x] LeakyReLU.cs
- [x] LogSigmoid.cs
- [x] LogSoftmax.cs
- [x] ReLU.cs
- [x] Sigmoid.cs
- [x] Softmax.cs
- [x] Tanh.cs
